core/storage/feedback/gae_models.py

```diff
@@ -155,6 +200,16 @@ class FeedbackMessageModel(base_models.BaseModel):
 
     @classmethod
     def _generate_id(cls, exploration_id, thread_id, message_id):
+        """Generates FeedbackMessageModel entry.
```

Comment:
Is it "Generates full message ID given the exploration ID, thread ID and message ID." ?